### PR TITLE
Revert "fix wrong link"

### PR DESCRIPTION
**Pull request by teoli2003**
Reverts mdn/content#33249

### CODE REVIEW

File: files/en-us/web/api/htmliframeelement/browsingtopics/index.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ browser-compat: api.HTMLIFrameElement.browsingTopics
 
 {{APIRef("HTML DOM")}}{{SeeCompatTable}}{{non-standard_header}}
 
-> **Warning:** This feature is currently opposed by two browser vendors. See the [Standards positions](/en-US/docs/Web/API/Topics_API#standards_positions) section below for details of opposition.
+> **Warning:** This feature is currently opposed by two browser vendors. See the [Standards positions](#standards_positions) section below for details of opposition.
 
-The **`browsingTopics`** property of the {{domxref("HTMLIFrameElement")}} interface is a boolean specifying that the selected topics for the current user should be sent with the request for the associated {{htmlelement("iframe")}}'s source in a {{httpheader("Sec-Browsing-Topics")}} header. This reflects the `browsingtopics` HTML attribute.
+The **`browsingTopics`** property of the {{domxref("HTMLIFrameElement")}} interface is a boolean specifying that that the selected topics for the current user should be sent with the request for the associated {{htmlelement("iframe")}}'s source in a {{httpheader("Sec-Browsing-Topics")}} header. This reflects the `browsingtopics` HTML attribute.
```
Comment (T34-active) [inline files/en-us/web/api/htmliframeelement/browsingtopics/index.md:16]:
```suggestion
The **`browsingTopics`** property of the {{domxref("HTMLIFrameElement")}} interface is a boolean specifying that the selected topics for the current user should be sent with the request for the associated {{htmlelement("iframe")}}'s source in a {{httpheader("Sec-Browsing-Topics")}} header. This reflects the `browsingtopics` HTML attribute.
```

File: files/en-us/web/api/htmliframeelement/browsingtopics/index.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ browser-compat: api.HTMLIFrameElement.browsingTopics
 
 {{APIRef("HTML DOM")}}{{SeeCompatTable}}{{non-standard_header}}
 
-> **Warning:** This feature is currently opposed by two browser vendors. See the [Standards positions](/en-US/docs/Web/API/Topics_API#standards_positions) section below for details of opposition.
+> **Warning:** This feature is currently opposed by two browser vendors. See the [Standards positions](#standards_positions) section below for details of opposition.
```
Comment (T34-active) [inline files/en-us/web/api/htmliframeelement/browsingtopics/index.md:14]:
```suggestion
> **Warning:** This feature is currently opposed by two browser vendors. See the [Standards positions](/en-US/docs/Web/API/Topics_API#standards_positions) for details of opposition.
```

File: files/en-us/web/api/htmliframeelement/browsingtopics/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/htmliframeelement/browsingtopics/index.md:16]: Not sure it is useful to change this here:

1. You created a PR that I merged
2. You asked for a revert as you want to continue to work on the original PR
3. I don't think we should make extra changes on this one, as you are creating a new PR.

As said, ideally, you create a follow-up PR for the precedence that I can review.

Also, if a PR is not ready for review, please create it as a draft.

Comment (T34-active) [reply files/en-us/web/api/htmliframeelement/browsingtopics/index.md:16]: i so sorry, Maybe I'm getting my own interpretation wrong. i will created new pull request for you. please delate this pull request

**Comment by teoli2003** [comment]
Sorry, I'm closing this.

You are redoing the very same change this PR was reverting. This PR is useless as these changes have already been merged.

For additional changes, just create a new PR.

**Pull request closed without merging**